README.md

```diff
@@ -47,7 +47,20 @@ With dependencies installed, execute the script as:
 ```
 lib-embedded-gcov/scripts/gcov_process_file.sh <build_directory> <binary_file>
 ```
+---
+### Memory output
+To retrieve coverage results through memory output, you'll need to extract information from the machine's RAM. The base pointer for this information is named gcov_output_buffer, and the index of the last byte of the report will be located at gcov_output_buffer + gcov_output_index at the end of execution. A recommended method for extracting this information is by using gdb and the following command sequence:
+
+```
+hbreak __gcov_exit
+continue
+finish
+dump memory memdump.bin gcov_output_buffer gcov_output_buffer+gcov_output_index
```

Comment:
These symbols are optimized-out when you build for performance. You can update your patch for `gcov_public.c` to add `__attribute__((used))` in these variables to prevent that.